partiql-playground/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
Getting a warning to add `LICENSE` file for this package 

```
[INFO]: License key is set in Cargo.toml but no LICENSE file(s) were found; Please add the LICENSE file(s) to your project directory
```

Reply:
Added.